app-utils/x11-ssh-askpass/autobuild/build

```diff
@@ -1,5 +1,10 @@
+abinfo "Configuring x11-ssh-askpass ..."
 ./configure ${AUTOTOOLS_DEF} ${AUTOTOOLS_AFTER}
```

Comment:
"$SRCDIR"/configure

Reply:
Also, are you sure that `${AUTOTOOLS_DEF[@]}` is not needed?

Reply:
Fixed